ibis/backends/dask/execution/arrays.py

```diff
@@ -0,0 +1,59 @@
+import itertools
+import operator
+
+import dask.dataframe as dd
+from dask.dataframe.groupby import SeriesGroupBy
+
+import ibis.expr.operations as ops
+from ibis.backends.pandas.execution.arrays import execute_node
+
+
+@execute_node.register(ops.ArrayLength, dd.Series)
+def execute_array_length(op, data, **kwargs):
+    return data.apply(len)
+
+
+@execute_node.register(ops.ArraySlice, dd.Series, int, (int, type(None)))
+def execute_array_slice(op, data, start, stop, **kwargs):
+    return data.apply(operator.itemgetter(slice(start, stop)))
+
+
+@execute_node.register(ops.ArrayIndex, dd.Series, int)
+def execute_array_index(op, data, index, **kwargs):
+    return data.apply(
+        lambda array, index=index: (
+            array[index] if -len(array) <= index < len(array) else None
+        )
+    )
+
+
+@execute_node.register(ops.ArrayConcat, dd.Series, (dd.Series, list))
+@execute_node.register(ops.ArrayConcat, list, dd.Series)
+def execute_array_concat(op, left, right, **kwargs):
+    return left + right
+
+
+@execute_node.register(ops.ArrayRepeat, dd.Series, dd.Series)
+@execute_node.register(ops.ArrayRepeat, int, (dd.Series, list))
+@execute_node.register(ops.ArrayRepeat, (dd.Series, list), int)
+def execute_array_repeat(op, left, right, **kwargs):
+    return left * right
+
+
+collect_list = dd.Aggregation(
```

Comment:
I found a bit weird to see this in the middle of bunch of rules. perhaps move to the top?